File: Manager/HagionSoft.TestDonkey.Android/app/src/main/java/com/hagionsoft/testdonkey/models/Notification.kt
package com.hagionsoft.testdonkey.models

import java.util.*

data class Notification(
    var TopicId: String = UUID.randomUUID().toString(),
    var TopicName: String,
    var TopicArn: String = "",
    var Cron: String,
    var Messages: MutableList<String> = mutableListOf())

File: Processor/HagionSoft.TestDonkey.AWSLambda/HagionSoft.TestDonkey.AWSLambda/Models/Notification.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace HagionSoft.TestDonkey.AWSLambda.Models
{
    public class Notification
    {
        public string TopicId { get; set; }
        public string TopicName { get; set; }
        public string TopicArn { get; set; }
        public string Cron { get; set; }
        public List<string> Messages { get; set; }
    }
}
